internal/rbd/controllerserver.go

```diff
@@ -468,11 +493,18 @@ func (cs *ControllerServer) createVolumeFromSnapshot(ctx context.Context, cr *ut
 	// as we are operating on single cluster reuse the connection
 	parentVol.conn = rbdVol.conn.Copy()
 
-	// create clone image and delete snapshot
-	err = rbdVol.cloneRbdImageFromSnapshot(ctx, rbdSnap, parentVol)
-	if err != nil {
-		util.ErrorLog(ctx, "failed to clone rbd image %s from snapshot %s: %v", rbdVol, rbdSnap, err)
-		return err
+	if rbdVol.ThickProvision {
+		err = parentVol.DeepCopy(rbdVol)
+		if err != nil {
+			return status.Errorf(codes.Internal, "failed to deep copy %q into %q: %w", parentVol, rbdVol, err)
```

Comment:
```suggestion
			return status.Errorf(codes.Internal, "failed to deep copy %q into %q: %v", parentVol, rbdVol, err)
```
Using `%w` inside status.Errorf throws `internal/rbd/controllerserver.go:499:11: printf: Errorf call has error-wrapping directive %w (govet)` govet linter error as `status.Errorf` internally calls `fmt.Sprintf()` 
https://github.com/grpc/grpc-go/blob/0257c8657362b76f24e7a8cfb61df48d4cb735d3/status/status.go#L62

Reply:
Its also the reason for `go-test` failure. @nixpanic ^^

Reply:
thanks! I'll test it out and verify that `make containerized-test` passes

Reply:
yesh.. its true, we dont need to wrap it.